src/app/zap-templates/templates/app/helper.js

```diff
@@ -714,6 +714,8 @@ function getPythonFieldDefault(type, options)
 function isWeaklyTypedEnum(label)
 {
   return [
+    "Enum8",
+    "Enum16",
```

Comment:
I don't think we want to do this, because the goal is for `isWeaklyTypedEnum` to go away altogether.

There are two separate concepts of weak-vs-strong typing for enums:

1. Is the type something that maps to an actual enum type as opposed to an integer)?  This is tested by `if_is_strongly_typed_chip_enum`.  This should be used any time we are looking at types.
2. For an enum definittion (which is by definition maps to an enum type) does it map to an enum type that is an enum class?  This is tested by `isWeaklyTypedEnum`.

Reply:
removed, thanks